packages/files-ui/src/Components/Modules/FileBrowsers/views/FilesList.tsx

```diff
@@ -800,8 +830,24 @@ const FilesList = ({ isShared = false }: Props) => {
       >
         {crumbs && moduleRootPath && (
           <Breadcrumb
-            crumbs={crumbs}
+            crumbs={crumbs.map((crumb, i) => ({
+              ...crumb,
+              component: (i < crumbs.length - 1) ? <FolderBreadcrumb
+                folderName={crumb.text}
+                onClick={crumb.onClick}
+                handleMove={(item) => {
+                  moveItems && crumb.path && moveItems(item.ids, crumb.path)
+                  setSelectedItems([])
+                }}
+                handleUpload={(item) => handleUploadOnDrop &&
+                  crumb.path &&
+                  handleUploadOnDrop(item.files, item.items, crumb.path)}
+              /> : null
+            }))
+            }
```

Comment:
Just a nit to have the ? and : at the beginning of the line, and closing all the } and ) on 1 line.
```suggestion
              component: (i < crumbs.length - 1)
                ? <FolderBreadcrumb
                  folderName={crumb.text}
                  onClick={crumb.onClick}
                  handleMove={(item) => {
                    moveItems && crumb.path && moveItems(item.ids, crumb.path)
                    setSelectedItems([])
                  }}
                  handleUpload={(item) => handleUploadOnDrop &&
                  crumb.path &&
                  handleUploadOnDrop(item.files, item.items, crumb.path)}
                />
                : null
            }))}
```